src/code42cli/cmds/auditlogs.py

```diff
@@ -34,14 +34,17 @@
 
 
 filter_option_usernames = click.option(
-    "--username", required=False, help="Filter results by usernames.", multiple=True,
+    "--actor", required=False, help="Filter results by actors.", multiple=True,
```

Comment:
should this maybe be `--actor-username` to match the `--affected-username/--affected-user-id` pattern and make explicit what value it expects?